src/TypeSpec.Extension/Emitter.Csharp/src/lib/client-model-builder.ts

```diff
@@ -230,16 +238,28 @@ export function createModelForService(
     function addChildClients(
         context: EmitContext<NetEmitterOptions>,
         client: SdkClient | SdkOperationGroup,
-        clients: InputClient[]
+        clients: InputClient[],
+        isAzureArm: boolean | undefined
     ) {
         const dpgOperationGroups = listOperationGroups(
             sdkContext,
             client as SdkClient
         );
         for (const dpgGroup of dpgOperationGroups) {
-            const subClient = emitClient(dpgGroup, client);
+            // For MPG, we skip the "Operations" operation group
```

Comment:
Other languages don't exclude these operation groups, such as java: [Operations](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/deviceregistry/azure-resourcemanager-deviceregistry/src/main/java/com/azure/resourcemanager/deviceregistry/implementation/OperationsClientImpl.java) and [OperationStatusClientImpl](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/deviceregistry/azure-resourcemanager-deviceregistry/src/main/java/com/azure/resourcemanager/deviceregistry/implementation/OperationStatusClientImpl.java)
So, this is specific for .NET.